src/WS.cpp

```diff
@@ -1217,26 +1351,63 @@ signed char WS::stream2_callback(struct lejp_ctx *ctx, char reason)
                 {
                 }
             }
-            append_message(
-                "%d", cfg->get<int>(u_ctx->path));
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
             break;
+        /*
         case PNT_STREAM2_JPEG_REFRESH:
             if (reason == LEJPCB_VAL_NUM_INT)
             {
                 if (cfg->set<int>(u_ctx->path, atoi(ctx->buf)))
                 {
                 }
             }
-            append_message(
-                "%d", cfg->get<int>(u_ctx->path));
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
+            break;
+        */
+        case PNT_STREAM2_FPS:
+            if (reason == LEJPCB_VAL_NUM_INT)
+            {
+                if (cfg->set<int>(u_ctx->path, atoi(ctx->buf)))
+                {
+                }
+            }
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
+            break;       
+        case PNT_STREAM2_JPEG_CHANNEL:
+            if (reason == LEJPCB_VAL_NUM_INT)
+            {
+                if (cfg->set<int>(u_ctx->path, atoi(ctx->buf)))
+                {
+                }
+            }
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
             break;
+        case PNT_STREAM2_STATS:
+            if (reason == LEJPCB_VAL_NULL)
+            {
+                uint8_t fps = 0;
+                uint32_t bps = 0;
+                if (u_ctx->root == "stream2")
```

Comment:
Why do we hardcode `stream2` here?

Reply:
in preparation to have more than 1 JPEG channel in future.